src/receivers/HTTPReceiver.spec.ts

```diff
@@ -504,11 +504,13 @@ describe('HTTPReceiver', function () {
 
         fakeReq.method = 'GET';
         receiver.requestListener(fakeReq, fakeRes);
-        assert(customRoutes[0].handler.calledWith({ ...fakeReq, params }, fakeRes));
+        let message = Object.assign(fakeReq, { params });
```

Comment:
Since this is a test and `message` is a value we expect and compare against, I think renaming it something more descriptive such as `expectedMessage` or something similar would provide more context

Let me know what you think

Reply:
That is much more clear, nice callout! I have gone and updated all of the tests with this 👍